lib/puppet/parser/functions/hiera_include.rb

```diff
@@ -39,7 +39,7 @@ module Puppet::Parser::Functions
   3. Like #1 but with all arguments passed in an array.
 
   More thorough examples of `hiera_include` are available at:
-  <http://docs.puppetlabs.com/hiera/1/puppet.html#hiera-lookup-functions>
+  <https://docs.puppetlabs.com/hiera/1/puppet.html#hiera-lookup-functions>
```

Comment:
Probably best to use this link now - https://docs.puppetlabs.com/hiera/3.0/puppet.html#hiera-lookup-functions

Example will need checked though.